test/e2e/persistent_volume/persistent_volume_test.go

```diff
@@ -0,0 +1,349 @@
+	// Create nfs provisioner
+	nfsProvisioner := &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      serverName,
+			Namespace: dev,
+			Labels: map[string]string{
+				"name": serverName,
+			},
+		},
+		Spec: appsv1.DeploymentSpec{
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					"name": serverName,
+				},
+			},
+			Template: v1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"name": serverName,
+					},
+				},
+				Spec: v1.PodSpec{
+					ServiceAccountName: svaName,
+					Containers: []v1.Container{
+						{
+							Name:  "nfs-provisioner",
+							Image: "openebs/provisioner-nfs:0.10.0",
+							Env: []v1.EnvVar{
+								{
+									Name: "NODE_NAME",
+									ValueFrom: &v1.EnvVarSource{
+										FieldRef: &v1.ObjectFieldSelector{
+											FieldPath: "spec.nodeName",
+										},
+									},
+								},
+								{
+									Name: "OPENEBS_NAMESPACE",
+									ValueFrom: &v1.EnvVarSource{
+										FieldRef: &v1.ObjectFieldSelector{
+											FieldPath: "metadata.namespace",
+										},
+									},
+								},
+								{
+									Name: "OPENEBS_SERVICE_ACCOUNT",
+									ValueFrom: &v1.EnvVarSource{
+										FieldRef: &v1.ObjectFieldSelector{
+											FieldPath: "spec.serviceAccountName",
+										},
+									},
+								},
+								{
+									Name:  "OPENEBS_IO_ENABLE_ANALYTICS",
+									Value: "true",
+								},
+								{
+									Name:  "OPENEBS_IO_NFS_SERVER_USE_CLUSTERIP",
+									Value: "true",
+								},
+								{
+									Name:  "OPENEBS_IO_INSTALLER_TYPE",
+									Value: "openebs-operator-nfs",
+								},
+								{
+									Name:  "OPENEBS_IO_NFS_SERVER_IMG",
+									Value: "openebs/nfs-server-alpine:0.10.0",
+								},
+							},
+							Resources: v1.ResourceRequirements{
+								Requests: v1.ResourceList{
+									"cpu":    resource.MustParse("50m"),
+									"memory": resource.MustParse("50M"),
+								},
+								Limits: v1.ResourceList{
+									"cpu":    resource.MustParse("200m"),
+									"memory": resource.MustParse("200M"),
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+
+	_, err := kClient.CreateDeployment(nfsProvisioner, dev)
+	Ω(err).NotTo(HaveOccurred())
+	ginkgo.By("Create nfs provisioner " + serverName)
+
+	// Create storage class
+	sc := &storagev1.StorageClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: scName,
+			Annotations: map[string]string{
+				"openebs.io/cas-type":   "nfsrwx",
+				"cas.openebs.io/config": "- name: NFSServerType\n  value: \"kernel\"\n- name: BackendStorageClass\n  value: \"standard\"\n",
+			},
+		},
+		Provisioner: "openebs.io/nfsrwx",
+	}
+
+	_, err = kClient.CreateStorageClass(sc)
+	Ω(err).NotTo(HaveOccurred())
+	ginkgo.By("Create storage class " + scName)
+}
+
+func deleteNfsRelatedRoles(serviceAccount string, clusterRole string, clusterRoleBinding string) error {
```

Comment:
Add a short logging about what the code is about to do here, eg `ginkgo.By("Deleting NFS related roles and bindings")`